components/service/digitalassetlinks/src/main/java/mozilla/components/service/digitalassetlinks/AndroidAssetFinder.kt

```diff
@@ -84,6 +85,7 @@ class AndroidAssetFinder {
         }
     }
 
+    @SuppressLint("PackageManagerGetSignatures")
     private fun PackageManager.getPackageSignatureInfo(packageName: String): PackageInfo? {
```

Comment:
Should we open an issue for this or the suppressing is just fine? I'm not completely aware of the context of the suppression but  the [lint description looks a bit spooky](https://android.googlesource.com/platform/tools/base/+/master/lint/libs/lint-checks/src/main/java/com/android/tools/lint/checks/GetSignaturesDetector.java#50)

Reply:
The issue mentioned was already fixed in Android 4+. I am not sure why the lint check didn't complain previously. In our cases we always look at all certificates, so that seems fine.